contrib/mesos/pkg/scheduler/meta/annotations.go

```diff
@@ -25,7 +25,6 @@ const (
 	TaskIdKey                = "k8s.mesosphere.io/taskId"
 	SlaveIdKey               = "k8s.mesosphere.io/slaveId"
 	OfferIdKey               = "k8s.mesosphere.io/offerId"
-	ExecutorIdKey            = "k8s.mesosphere.io/executorId"
```

Comment:
I would leave this as it is, i.e. with removed ExecutorIdKey. With @sur's roles patch we get an ExecutorInfo cache anyway which gives us back everything we would need for framework messages.